### PR TITLE
Don't use old tool to run `meteor update` in an RC

### PR DESCRIPTION
**Pull request by dgreensp**
See commit

### CODE REVIEW

File: tools/cli/commands-packages.js [hunk 1]
```diff
@@ -1270,6 +1270,26 @@ var maybeUpdateRelease = function (options) {
           releaseTrack + ".");
       return 1;
     }
+
+    if (release.current && options.appdir && ! options.patch) {
```
Comment (glasser) [inline tools/cli/commands-packages.js]: `appDir`

**Comment by glasser** [comment]
ignoring the fact that your code never ran, it seems reasonable.

maybe you want to print a message if you take that path though?

**Comment by dgreensp** [comment]
Thanks for reading.  A message might be good, yeah.

Is `release.current` the project version?  We don't actually look at the project version until later in the function, but given the set of checks around this code, I can't think of other scenarios.

I'll probably just publish an RC to test; I can't think of another way.

**Comment by glasser** [comment]
It is the currently running release. It is calculated laboriously in main.js.  Basically it's (when not running from a checkout):

- Is the METEOR_SPRINGBOARD_RELEASE env var set? (This is done basically by the code you're looking at here and only a couple more places.)  If so, use that.  Set `release.forced` to true.

- Otherwise, was `--release foo` specified on the command line? If so, use that. Set `release.forced` and `release.explicit` to true.

- Otherwise, are we in an app? Use the app's release.

- Otherwise use the most recent recommended release on the default (METEOR) track.

In the particular part of the code you're looking at, we already are in a `! release.forced` conditional, so release.current is either the app release if `options.appDir` or the latest recommended release.

Oh, there is one super weird special case where if you're running a built meteor but the `.meteor/release` file says "none" (which is what you have for an app created by a checkout version), `release.current` is actually null.  Super weird I know.  Not sure how far that gets before turning into an error if it does...

**Comment by dgreensp** [comment]
Merged to `release-1.2`.

**Pull request closed without merging**